### PR TITLE
I wrote a tests login general

### PR DESCRIPTION
**Pull request by ikonna**

### CODE REVIEW

File: test/selectors/login.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default {
+    loginButton: '//*[@id="login"]',
```
Comment (artsenius) [inline test/selectors/login.js]: please don't use Xpath.
Only if you have tried the other approaches and they don't work

Comment (ikonna) [reply test/selectors/login.js]: Can u show me how to do with selectors?

**Comment by amadjouga** [comment]
@artsenius Do we need to create variables or we can just put all parameters to the assert like in this pull request?

File: test/selectors/login.js (hunk 1 above)
Comment (Miksyuk) [reply test/selectors/login.js]: i can show but it will be tomorrow

Comment (ikonna) [reply test/selectors/login.js]: oh Thanks I appreciate it Sergei

**Comment by artsenius** [comment]
> @artsenius Do we need to create variables or we can just put all parameters to the assert like in this pull request?

If it's short, you can use it directly in assert. But if it's too long, then it's use variable in order to avoid too long lines.

**Comment by artsenius** [comment]
@vianat - I can help you tomorrow to do this.

**Pull request merged**